main.go

```diff
@@ -309,7 +309,9 @@ func main() {
 	cmdVersion.PersistentFlags().StringVarP(&versionOutput, "output", "o", versionOutput, "Output format; available options are 'json'")
 
 	rootCmd := &cobra.Command{
-		Use: "knest",
+		Use:           "knest",
+		SilenceUsage:  true,
+		SilenceErrors: true,
```

Comment:
We should still print errors?

Reply:
for example when knest call kubectl command, if kubectl return an error, kubectl will print it, and knest will print it again, with `SilenceErrors` only print the error once.

```
Error from server (NotFound): clusters.cluster.x-k8s.io "abc" not found
delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
exit status 1
```

```
Error from server (NotFound): clusters.cluster.x-k8s.io "abc" not found
Error: delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
exit status 1
```

Reply:
What about errors from knest?

Reply:
the repeated print are caused by https://github.com/smartxworks/knest/blob/main/main.go#L322, maybe we should keep `SilenceErrors=false`, and remove https://github.com/smartxworks/knest/blob/main/main.go#L322

Reply:
example logs:
```
Error from server (NotFound): clusters.cluster.x-k8s.io "abc" not found
Error: delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
exit status 1
```

```
Error from server (NotFound): clusters.cluster.x-k8s.io "abc" not found
Error: delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
delete cluster CR: run command "/usr/local/bin/kubectl delete clusters.cluster.x-k8s.io abc --namespace default --wait": exit status 1
exit status 1
```

Reply:
👌Let's do that